src/store/index.ts

```diff
@@ -680,6 +681,35 @@ export class Main extends VuexModule {
     await this.syncConfiguration();
   }
 
+  @Action
+  async toggleGlobalZMaxMerge() {
+    const layers = this.configuration?.view?.layers;
+    if (!layers || !this.dataset) {
+      return;
+    }
+    const currentZMaxMerge = layers.every(
+      layer => layer.z.type === "max-merge"
+    );
+    const newLayerZ: IDisplaySlice = currentZMaxMerge
+      ? { type: "current", value: null }
+      : { type: "max-merge", value: null };
+    layers.every(layer => (layer.z = newLayerZ));
```

Comment:
Why don't you use a forEach loop here?
In my opinion, every method exists to check if every element of an array pass a test

Reply:
Yes sorry, I copy pasted something and it went wrong